packages/xo-web/src/common/xo/index.js

```diff
@@ -53,6 +53,11 @@ import parseNdJson from './_parseNdJson'
 
 // ===================================================================
 
+export const DEFAULT_ITEMS_PER_PAGE = 20
```

Comment:
We should probably keep 10 as the default value, otherwise it would change for everybody.